rogue.py

```diff
@@ -130,3 +130,30 @@ def __init__(self, level):
             feats=[ASI([["dex", 2]]), ASI([["dex", 1]])],
             base_feats=base_feats,
         )
+
+class ArcaneTricksterRogue(Character):
```

Comment:
Is it possible to unify the two classes similar to Fighter?

Reply:
It's tricky. I want the weapons to be reusable within `Rogue`, but I also need the individual subclasses to access them before initializing `Rogue` as they have customized attack patterns, and I have to have all subclass feats defined before I pass initialization up to the parent class. How do you envision this design working?